diff --git a/src/conn_handler.c b/src/conn_handler.c
index 8e77cb3..41bc63d 100644
--- a/src/conn_handler.c
+++ b/src/conn_handler.c
@@ -272,7 +272,7 @@ static int handle_ascii_client_connect(statsite_conn_handler *handle) {
                 default:
                     type = UNKNOWN;
                     syslog(LOG_WARNING, "Received unknown metric type! Input: %c", *type_str);
-                    close_client_connection(handle->conn);
+                    //close_client_connection(handle->conn);
                     if (should_free) free(buf);
                     return -1;
             }
@@ -298,14 +298,14 @@ static int handle_ascii_client_connect(statsite_conn_handler *handle) {
                 metrics_add_sample(GLOBAL_METRICS, type, buf, val);
             } else {
                 syslog(LOG_WARNING, "Failed value conversion! Input: %s", val_str);
-                close_client_connection(handle->conn);
+                //close_client_connection(handle->conn);
                 if (should_free) free(buf);
                 return -1;
 
             }
         } else {
             syslog(LOG_WARNING, "Failed parse metric! Input: %s", buf);
-            close_client_connection(handle->conn);
+            //close_client_connection(handle->conn);
             if (should_free) free(buf);
             return -1;
         }
